GeoFire.podspec

```diff
@@ -1,15 +1,15 @@
 Pod::Spec.new do |s|
   s.name         = "GeoFire"
-  s.version      = "2.0.0"
+  s.version      = "2.0.1"
```

Comment:
GeoFire is already on 4.x.

I'm not opposed to making this a major version bump, but I'd do it to prevent from auto-updates causing symbol collisions, rather than changes in DB behavior (which shouldn't exist).